### DISCUSSION TITLE
posts/key-solving-problem

### DISCUSSION THREAD

**Discussion by giscus** [seongminn/soynoah.me#5] [Announcements]
# posts/key-solving-problem

훕치치 3차 릴리즈를 마치며 느낀 점을 중심으로,

https://seongmin-v2.vercel.app/posts/key-solving-problem

**Comment by Jin409** [comment]
회고에 대해서 피드백 진행하기 좋은 의견이네용
팀장으로서 한번 검토해보도록 하찌요.
